tui/features.sh

```diff
@@ -12,7 +12,18 @@ if [[ "$RASPBERRYPI_MODEL" != *"Raspberry Pi 3"* ]] && [[ "$KERNEL" != *"microso
 fi
 
 OVOS_FEATURES=$(whiptail --separate-output --title "$TITLE" \
-  --checklist "$CONTENT" --cancel-button "$CANCEL_BUTTON" --ok-button "$OK_BUTTON" --yes-button "$OK_BUTTON" "$TUI_WINDOW_HEIGHT" "$TUI_WINDOW_WIDTH" "${#features[@]}" "${features[@]}" 3>&1 1>&2 2>&3)
+  --checklist "$CONTENT" --cancel-button "$BACK_BUTTON" --ok-button "$OK_BUTTON" --yes-button "$OK_BUTTON" "$TUI_WINDOW_HEIGHT" "$TUI_WINDOW_WIDTH" "${#features[@]}" "${features[@]}" 3>&1 1>&2 2>&3)
+
+exit_status=$?
+
+if [ $exit_status != 0 ]; then
```

Comment:
The `"` are missing around `$exist_status` and `-ne` should be used instead of `!=`.